examples/nextjs/src/app/_components/button.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
+
+import { useLogger } from '@/lib/axiom/client';
+
+export default function Button() {
+  const logger = useLogger();
```

Comment:
maybe move onClick handler to the component body so the logger usage is more visible.

Reply:
Fixed it!